src/Processors/Formats/Impl/AvroRowInputFormat.h

```diff
@@ -155,7 +155,7 @@ class AvroConfluentRowInputFormat final : public IRowInputFormat
     std::unordered_map<SchemaId, AvroDeserializer> deserializer_cache;
     const AvroDeserializer & getOrCreateDeserializer(SchemaId schema_id);
 
-    avro::InputStreamPtr input_stream;
+    /// avro::InputStreamPtr input_stream; /* proton: updated */
```

Comment:
I am thinking eliminating them all for deletion since we know what we are doing to do and we are doing better than CH community.